velox/exec/DistinctAggregations.cpp

```diff
@@ -211,6 +211,7 @@ std::unique_ptr<DistinctAggregations> DistinctAggregations::create(
   }
 
   const auto type = inputType->childAt(aggregates[0]->inputs[0]);
+  VELOX_CHECK(type->isComparable());
```

Comment:
Let's add this check to AggregationPlanNode, make it USER_CHECK and provide a clear error message. Let's make these changes in a separate PR.

Reply:
OK, got it.